src/adagrams.js

```diff
@@ -1,15 +1,155 @@
+// add export to run tests!
 export const drawLetters = () => {
+  // const drawLetters = () => {
   // Implement this method for wave 1
+
+  const availableLetters = {
```

Comment:
This can be declared outside of the function so it doesn't have to be recreated each time the function is called.